Source/Cotter.swift

```diff
@@ -244,8 +255,8 @@ public class Cotter {
         configuration: [String:Any] = [:]
     ) {
         print("configuring Cotter's object...")
-        CotterAPIService.shared.baseURL = URL(string: "https://www.cotter.app/api/v0")!
-//        CotterAPIService.shared.baseURL = URL(string: "http://localhost:1234/api/v0")!
+//        CotterAPIService.shared.baseURL = URL(string: "https://www.cotter.app/api/v0")!
```

Comment:
Wait, shouldn't the `CotterAPIService.shared.baseURL` variable be the prod URL instead of the localhost one?